data_models/website.py

```diff
@@ -0,0 +1,35 @@
+"""This script creates a class to store
+information about to a website"""
+
+import validators
+
+from data_models.contents import Contents
+
+
+class Website:
+    """This class stores the information about a website"""
+
+    __LOGO_API = 'https://www.google.com/s2/favicons?domain='
+
+    def __init__(self, url):
+        self.url = url
+        self.is_valid = validators.url(self.url) is True
+        self.domain = self.url.split('/')[2] if self.is_valid else None
+        self.logo_url = self.__LOGO_API + self.domain if self.domain else None
+        self.contents = Contents()
+
+    def set_contents(self, contents):
+        self.contents = contents
+
+    def get_formatted_dict(self):
+        """This method converts the Website object into dictionary"""
+
+        _website_details = dict()
+
+        for attr in vars(self):
+            if attr == 'contents':
+                _website_details[attr] = vars(self)[attr].get_formatted_list()
+            else:
+                _website_details[attr] = vars(self)[attr]
+
+        return _website_details
```

Comment:
Can you override `__dict__` here and in Contents to generalise it?

Reply:
Updated accordingly!